Generals/Code/GameEngine/Source/GameLogic/Object/Update/ParticleUplinkCannonUpdate.cpp

```diff
@@ -452,8 +455,7 @@ UpdateSleepTime ParticleUplinkCannonUpdate::update()
 				break;
 		}
 
-		Drawable *beam = TheGameClient->findDrawableByID( m_orbitToTargetBeamID );
-		if( beam && orbitalBirthFrame <= now && now <= orbitalDeathFrame )
+		if( orbitalBirthFrame <= now && now < orbitalDeathFrame )
```

Comment:
I had to change this condition from `now <= orbitalDeathFrame` to `now < orbitalDeathFrame`.

`now <= orbitalDeathFrame` is slightly incorrect and mismatches with retail (when the drawable test is gone), because it contradicts the condition above:

```cpp
case LASERSTATUS_DECAYING:
{
  if( orbitalDeathFrame <= now )
```